worker/uniter/runner/runner.go

```diff
@@ -369,6 +381,7 @@ func (runner *runner) runCharmHookOnRemote(hookName string, env []string, charmL
 	// When running an action, We capture stdout and stderr
 	// separately to pass back.
 	var actionErr = actionOut
+	var hookErrLogger *charmrunner.HookLogger
```

Comment:
all the two loggers are already stopped here in defer, no need to pass into `ExecParams` to call stop inside `executor`.

Reply:
as discussed, we can't combine them. so all good.